config/sync/core.entity_form_display.node.author.default.yml

```diff
@@ -0,0 +1,54 @@
+uuid: 0f1712d5-b377-47a2-9e20-18376f94bdee
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.author.field_a_biography
+    - field.field.node.author.field_a_image
+    - field.field.node.author.field_a_name
```

Comment:
You don't need to create a new name field, you can reuse the title field that comes with the content type.